src/odemis/driver/test/orsay_test.py

```diff
@@ -31,6 +31,10 @@
 from odemis.driver import orsay
 from odemis.model import HwError
 from odemis import model
+from odemis.util import timeout
+
+logging.getLogger().setLevel(logging.DEBUG)
+logging.basicConfig(format="%(asctime)s  %(levelname)-7s %(module)s:%(lineno)d %(message)s")
 
 TEST_NOHW = (os.environ.get("TEST_NOHW", "0") != "0")  # Default to Hw testing
```

Comment:
I think this whole part is broken. It converts to a boolean, and then it's compared to a string.
I'd suggest to copy the same as in `xt_client_test.py`:
```python
TEST_NOHW = os.environ.get("TEST_NOHW", "0")  # Default to Hw testing
if TEST_NOHW == "sim":
    pass
elif TEST_NOHW == "0":
    TEST_NOHW = False
elif TEST_NOHW == "1":
    TEST_NOHW = True
else:
    raise ValueError("Unknown value of environment variable TEST_NOHW=%s" % TEST_NOHW)
```

In addition, there are several checks like `if TEST_NOHW == 1`. It can never happen. It should be just `if TEST_NOHW`.